### ISSUE TITLE
Change CDB configuration to support mutliple scenarios

### ISSUE THREAD

**Issue by acaproni** [IntegratedAlarmSystem-Group/ias#2]
The ID of the IAS is an integer automatically generated by the DB. Other DB entities do not refer to the IAS (i.e. no foreign key) because there is only one instance of the IAS running at a given time.

By changing the ID of the IAS to a user provided string and adding foreign keys to the other tables, it would be possible to have more then one configured IAS in the database. At run time, it would be possible to select which IAS configuration to run providing the IAS identifier as  a java property or an environment variable. The simplest use case is to have one IAS for testing and one for production but this feature could be useful in operation to have for example different deployments.

**Comment by sfehlandt** [comment]
Maybe we can use different CDB's instantiations for different IAS instantiations.
This way we can have a real deployment in parallel to a development deployment, each with different levels of completion and separate databases to avoid conflicts between different versions of the IAS

**Comment by acaproni** [comment]
Yes Sebastian, I would keep your suggestion and go for different instantiations at least in the beginning.
Note that it is also possible to define the CDB by means of JSON files, intended mostly for testing.
